osquery/tables/system/windows/windows_eventlog.cpp

```diff
@@ -87,61 +85,91 @@ Status parseWelXml(QueryContext& context, std::wstring& xml_event, Row& row) {
 void renderQueryResults(QueryContext& context,
                         EVT_HANDLE queryResults,
                         RowYield& yield) {
-  std::vector<EVT_HANDLE> events(kNumEventsBlock);
-  unsigned long numEvents = 0;
-
-  // Retrieve the events one block at a time
-  auto ret = EvtNext(
-      queryResults, kNumEventsBlock, events.data(), INFINITE, 0, &numEvents);
-  while (ret != FALSE) {
-    for (unsigned long i = 0; i < numEvents; i++) {
-      unsigned long renderedBuffSize = 0;
-      unsigned long renderedBuffUsed = 0;
-      unsigned long propCount = 0;
-      if (!EvtRender(nullptr,
-                     events[i],
-                     EvtRenderEventXml,
-                     renderedBuffSize,
-                     nullptr,
-                     &renderedBuffUsed,
-                     &propCount)) {
-        if (GetLastError() != ERROR_INSUFFICIENT_BUFFER) {
-          LOG(WARNING) << "Failed to get the size of rendered event "
+  uint32_t kNumEventsBlock = 1024;
```

Comment:
nitpick, remove the `k` naming here, that should be reserved for global const variables.